### PR TITLE
Support high intensity ANSI colors #16 with jansi-1.12

### PR DESCRIPTION
**Pull request by marlene01**
pom.xml changes: - increased dependency of jansi to 1.12
                 - fixed the pom error with animal-sniffer plugin
AnsiHtmlOutputStream changes:
 - added method processSetForegroundColor with additional parameter boolean bright to handle the 90 to 97 colors.

### CODE REVIEW

**Comment by dblock** [comment]
There're a lot of spaces -> tabs changes here, can you please fix that? Otherwise we have huge diffs and can't figure out what's going on.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ Next Release
 ============
 
 * Your contribution here.
+* [#72](https://github.com/dblock/jenkins-ansicolor-plugin/pull/72): Support high intensity ansi colors - [@marlene01](https://github.com/marlene01).
```
Comment (dblock) [inline CHANGELOG.md]: ansi => ANSI

**Comment by dblock** [comment]
This needs tests, please, at least around `processSetForegroundColor` functionality.

**Comment by marlene01** [comment]
I fixed the issues in a new branch called 16_high_intensity_ansi_colors_one_commit which can be found here:
[https://github.com/marlene01/jenkins-ansicolor-plugin/tree/16_high_intensity_ansi_colors_one_commit](https://github.com/marlene01/jenkins-ansicolor-plugin/tree/16_high_intensity_ansi_colors_one_commit)

**Comment by dblock** [comment]
Merged, thank you! We should do a release soon.

**Comment by marlene01** [comment]
Great! :-)
But what about the dependency in jenkins-core to jansi 1.9?
Should I create a Issue on the [https://issues.jenkins-ci.org/browse/JENKINS-35254?jql=project%20%3D%20JENKINS](https://issues.jenkins-ci.org/browse/JENKINS-35254?jql=project%20%3D%20JENKINS) board?

**Comment by dblock** [comment]
Definitely @marlene01, at the very least Jenkins future versions should upgrade Jansi.

**Comment by dblock** [comment]
This was finally released in 0.4.3. Thanks for your patience.

**Pull request merged**